.spectral.yaml

```diff
@@ -27,15 +27,15 @@ rules:
         min: 1
         max: 1
 
-  alchemy-servers-url:
-    severity: error
-    description: Url must be valid format.
-    given: $.servers.*
-    then:
-      field: url
-      function: pattern
-      functionOptions:
-        match: "https://{network}.g.alchemy.com/v2"
+  # alchemy-servers-url:
+  #   severity: error
+  #   description: Url must be valid format.
+  #   given: $.servers.*
+  #   then:
+  #     field: url
+  #     function: pattern
+  #     functionOptions:
+  #       match: "https://{network}.g.alchemy.com/v2"
```

Comment:
@bmoyroud context here?

Reply:
current validation does not work for if you use https://astar-mainnet.g.alchemy.com/v2 rather than https://{network}.g.alchemy.com/v2 as server url

actually will comment out if adding testnet